package.json

```diff
@@ -35,10 +35,10 @@
   },
   "peerDependencies": {
     "cldr-data": ">=25",
-    "globalize": ">=1.1.0-rc.5"
+    "globalize": "^1.1.0"
   },
   "devDependencies": {
-    "babel": "^5.6.14",
+    "babel": "^6.3.26",
```

Comment:
@ashleygwilliams Any suggestions? I feel like you recently had a tweet about something like this with Travis builds.